src/sdk/namespace_blob.js

```diff
@@ -12,7 +12,9 @@ const azure_storage = require('../util/new_azure_storage_wrap');
 const stream_utils = require('../util/stream_utils');
 const s3_utils = require('../endpoint/s3/s3_utils');
 const schema_utils = require('../util/schema_utils');
-const valid_attr_regex = /^[A-Za-z_][A-Za-z0-9_]+$/;
+const invalid_azure_tag_regex = /[^a-zA-Z0-9_]/;
```

Comment:
```suggestion
const invalid_azure_md_regex = /[^a-zA-Z0-9_]/;
```

Reply:
We are currently not supporting tagging in namespace_blob (see Tagging section in this file), I find it very confusing to keep using tag term when manipulating metadata.